test/client/form-cache.spec.js

```diff
@@ -275,5 +275,87 @@ describe('Client Form Cache', () => {
                 })
                 .then(done, done);
         });
+
+        describe('form media (only) cache updates', () => {
+            let resultSurvey;
+
+            /** @type {SinonSpy} */
+            let storeSurveyUpdateSpy;
+
+            beforeEach((done) => {
+                getFileSpy.restore();
+                getFileSpy = sandbox
+                    .stub(connection, 'getMediaFile')
+                    .callsFake(() => Promise.reject(new Error('Fail!')));
```

Comment:
`.stub().throws()` doesn't work with the original pre-PR code. The reason it would work here is probably because I used await without try and catch (and throwing). I like to be able to run the tests on the old code, so used callsFake instead.